apps/coscrad-frontend/src/utils/custom-hooks/use-id-from-location.ts

```diff
@@ -0,0 +1,14 @@
+import { useParams } from 'react-router-dom';
+import { ID_ROUTE_PARAM_KEY } from '../../app/routes';
+
+export const useIdFromLocation = () => {
+    const paramsFromLocation = useParams();
+
+    const idFromLocation = paramsFromLocation[ID_ROUTE_PARAM_KEY];
+
+    if (typeof idFromLocation !== 'string' || idFromLocation.length === 0) {
+        throw new Error(`Failed to read ID from route params`);
+    }
+
+    return [idFromLocation] as const;
```

Comment:
We should consider dropping the tuple return type. Originally, I thought it was convention to return a tuple from a custom hook. But thinking of `useDispatch` I realize it's not necessary when you know that you'll only ever return one thing.